modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/js/ddm_form.js

```diff
@@ -1212,6 +1213,47 @@ AUI.add(
 						);
 				},
 
+				syncTranslatedLabelsUI(locale) {
+					const instance = this;
+
+					const defaultLocale = instance.getDefaultLocale();
+
+					if (locale === defaultLocale) {
+						return;
+					}
+
+					const localizationMap = instance.get('localizationMap');
+
+					const labelItem = A.one(
+						'#_com_liferay_dynamic_data_lists_web_portlet_DDLPortlet_PaletteContentBox a[data-value="' +
```

Comment:
I'm not sure if this could be retrieved from some properties or a constant?

Reply:
I get this namespace dinamically:

`const regexpFieldsNamespace = new RegExp(
instance.get('name') +
	INSTANCE_ID_PREFIX +
	instance.get('instanceId'),
'gi'
);
const fieldsNamespace = instance
.getInputName()
.replace(regexpFieldsNamespace, '');

const labelItem = A.one(
'#' +
	fieldsNamespace +
	'PaletteContentBox a[data-value="' +
	locale +
	'"] .label'
);`